products/godot.md

```diff
@@ -11,6 +11,7 @@ changelogTemplate: |
 eolColumn: Critical, Security and Platform support
 activeSupportColumn: true
 releaseDateColumn: true
+versionCommand: godot --version
```

Comment:
Must be moved before the `releasePolicyLink` as per [our style guide](https://github.com/endoflife-date/endoflife.date/wiki/Style-Guide).